presto-main/src/main/java/com/facebook/presto/operator/scalar/FormatUtilsFunctions.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.operator.scalar;
+
+import com.facebook.presto.spi.function.Description;
+import com.facebook.presto.spi.function.ScalarFunction;
+import com.facebook.presto.spi.function.SqlType;
+import com.facebook.presto.spi.type.StandardTypes;
+import io.airlift.slice.Slice;
+import io.airlift.slice.Slices;
+
+import java.math.RoundingMode;
+import java.text.DecimalFormat;
+
+import static com.facebook.presto.spi.StandardErrorCode.NUMERIC_VALUE_OUT_OF_RANGE;
+import static com.facebook.presto.util.Failures.checkCondition;
+import static java.lang.Float.intBitsToFloat;
+import static java.lang.String.format;
+
+public final class FormatUtilsFunctions
+{
+    private FormatUtilsFunctions() {}
+
+    @Description("format to human readable count value")
+    @ScalarFunction("format_count")
+    @SqlType(StandardTypes.VARCHAR)
```

Comment:
`BIGINT` has a limit, it's `Long.MAX_VALUE`, which makes it expressable by `VARCHAR(19)`.